### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have created a Product Page for my eCommerce app.
Please review and suggest.

Note: Click on any of the Navbar items (Dining, Decor, Kitchen) to view products page

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Code is clean.

**Review by VLeads — APPROVED** [review]
UI is also great. Go ahead :)

File: css/product.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+.checkbox-field,
+.radio-field {
+  width: 1.15rem;
+  height: 1.15rem;
+  position: relative;
+  top: 0.15rem;
+}
+
+/* ------------------- Wishlst Icon- -------------------- */
```
Comment (foolhardy21) [inline css/product.css]: all looks good to me. just this spelling :)

**Review by foolhardy21 — APPROVED** [review]
good to merge.

**Review by BalajiHarinath — APPROVED** [review]
UI is clean and code looks good as well.

File: css/product.css (hunk 1 above)
Comment (ApurvaSawant11) [reply css/product.css]: Fixed it. Thanks

**Pull request merged**